native/chat/chat.react.js

```diff
@@ -148,17 +153,37 @@ export type ChatNavigationProp<
   RouteName: $Keys<ChatParamList> = $Keys<ChatParamList>,
 > = ChatRouterNavigationProp<ScreenParamList, RouteName>;
 
+export type ChatTopTabsNavigationProp<
+  RouteName: $Keys<ChatTopTabsParamList> = $Keys<ChatTopTabsParamList>,
+> = ChatRouterNavigationProp<ScreenParamList, RouteName>;
```

Comment:
`ChatRouterNavigationProp` is the wrong type here, you should be importing `MaterialTopTabNavigationProp`. [This](https://medium.com/@ashoat_t/flow-libdefs-for-react-navigation-5-7dcf1f9da8b7) might be worth a read